### DISCUSSION TITLE
Adding to Conda

### DISCUSSION THREAD

**Discussion by WilliamParsonsV** [CCSDSPy/ccsdspy#66] [Q&A] [answered]
I'm using a Conda environment to run my scripts and I have a requirements.txt file for auto-installing dependencies. I have to manually download pip and then pip install CCSDSPy as it doesn't seem to be on Conda or Conda-forge. I'm not sure how difficult it is to add packages/libraries to Conda but it would certainly be a welcome addition!

**Comment by ddasilva** [comment, marked as answer]
Hi @WilliamParsonsV . You can add pip dependencies to conda `environment.yml` files, and they will get installed automatically when you make the conda environment. Here is an example from one of my projects. Does this solve your problem?

```yaml
name: myproject
channels:
  - conda-forge
dependencies:
  # Base python
  - python>=3.8.5
  # Conda dependencies
  - astropy==5.0
  - termcolor==1.1.0
  - xmltodict==0.12.0
  # Pip Dependencies
  - pip
  - pip:
      - ccsdspy

**Reply by WilliamParsonsV** [reply]
It does, thank you!!
